examples/tutorials/01_basics/07_backpack_lights.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2016 Anki, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the file LICENSE.txt or at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Control Cozmo's Backpack lights
+
+This script shows how you can control Cozmo's backpack lights and set
+them to different colors.
```

Comment:
The only other constant we have in SDK is for white. I actually have a small change that will follow this one to add a use of that here on a new "center lights only" version of that function which avoids the issue of the left/right LEDs being red-only.